src/models/token.js

```diff
@@ -0,0 +1,37 @@
+const { Sequelize, DataTypes } = require('sequelize');
+
+const { User } = require('./user');
+const { sequelize } = require('../utils/db.js');
+
+const Token = sequelize.define(
+  'Token',
+  {
+    refreshToken: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    expireAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: Sequelize.literal(`NOW() + INTERVAL 30 DAYS`),
```

Comment:
Using `Sequelize.literal` with a raw SQL expression for `defaultValue` may not be supported by all databases. Consider using JavaScript Date manipulation or Sequelize hooks to set the default expiration date.